services/galley/src/Galley/API/Cells.hs

```diff
@@ -49,4 +49,4 @@ instance HasCellsState () where
 
 shouldPushToCells :: (HasCellsState a) => a -> EventType -> Bool
 shouldPushToCells st et =
-  isCellsConversationEvent et && getCellsState st == CellsReady
+  isCellsConversationEvent et && getCellsState st /= CellsDisabled
```

Comment:
```suggestion
  isCellsConversationEvent et && case getCellsState st of
    CellsDisabled -> False
    CellsPending -> True
    CellsReady -> True
```

(to make changes in the state type cause type errors in this place if it's overlooked)